src/zeebe/lib/interfaces-grpc-1.0.ts

```diff
@@ -506,6 +506,45 @@ export interface ThrowErrorRequest {
 export interface CompleteJobRequest<Variables = IProcessVariables> {
 	readonly jobKey: string
 	variables: Variables
+	/** Since Camunda 8.7 */
+	result?: JobResult
+}
+
+/**
+ * Since Camunda 8.7
+ */
+export interface JobResult {
+	// Indicates whether the worker denies the work, i.e. explicitly doesn't approve it.
+	// For example, a Task Listener can deny the completion of a task by setting this flag to true.
+	// In this example, the completion of a task is represented by a job that the worker can complete as denied.
+	// As a result, the completion request is rejected and the task remains active.
+	// Defaults to false.
+	denied?: boolean
+	// Attributes that were corrected by the worker.
+	// The following attributes can be corrected, additional attributes will be ignored:
+	//   * `assignee` - reset by providing an empty String
+	//   * `dueDate` - reset by providing an empty String
+	//   * `followUpDate` - reset by providing an empty String
+	//   * `candidateGroups` - reset by providing an empty list
+	//   * `candidateUsers` - reset by providing an empty list
```

Comment:
🔧 The language in this documentation was changed for clarity. Would be good to take along in this sdk as well: `reset` has become `clear` for the Job Result corrections

```suggestion
	//   * `assignee` - clear by providing an empty String
	//   * `dueDate` - clear by providing an empty String
	//   * `followUpDate` - clear by providing an empty String
	//   * `candidateGroups` - clear by providing an empty list
	//   * `candidateUsers` - clear by providing an empty list
```